packaging/CPackDebUploadPPA.cmake

```diff
@@ -46,7 +46,7 @@ endif()
 # dpkg --compare-versions 2.4.0~alpha~pre1~git7859  lt 2.4.0~alpha1~5463~gf2da9e619d && echo true
 # dpkg --compare-versions 2.4.0~alpha1~5463~gf2da9e619d lt 2.4.0 && echo true
```

Comment:
The comment still mentions alpha1?

Please also fix the commit message -> "bump"